goat

```diff
@@ -151,9 +160,27 @@ link()
 		rm -- "$GOAT_PATH/$1"
 	fi
 
-	ln -fs -- "$targetpath" "$GOAT_PATH/$1"
+	$LN "$targetpath" "$GOAT_PATH/$1"
 }
 
+unix_ln()
+{
+	targetpath="$1"
+	destination="$2"
+	ln -fs -- "$targetpath" "$destination"
+}
+
+mingw_ln()
+{
+	targetpath="$1"
+	destination="$2"
+	#cmd "$destination" "$targetpath"
+	"$GOAT_PATH/mingw_ln.bat" "$destination" "$targetpath"
```

Comment:
I do not use Windows so I assume that those bat scripts need neither an explicit interpreter nor an execute bit. Is that right?

Reply:
Nope! Because of MSYS magic, you can also call it as `MiNGW_LN.bAt` and expect it to work.

Reply:
Wonderful!